sdk/scheduler/src/test/java/com/mesosphere/sdk/scheduler/DefaultSchedulerTest.java

```diff
@@ -190,6 +192,26 @@ public boolean supportsDomains() {
     private ConfigStore<ServiceSpec> configStore;
     private DefaultScheduler defaultScheduler;
 
+    @BeforeClass
+    public static void beforeAll() {
+        // Disable background TaskKiller thread, to avoid erroneous kill invocations
+        try {
+            TaskKiller.reset(false);
+        } catch (Exception e) {
+            throw new IllegalStateException(e);
+        }
+    }
+
+    @AfterClass
+    public static void afterAll() {
+        // Re-enable TaskKiller thread
+        try {
+            TaskKiller.reset(false);
```

Comment:
The comment states that this is being re-enabled, but the code is the same for disabling above. Either one of the calls or the comments are incorrect.

Reply:
Whoops, yes. Fixed